src/components/subscriptions/licenses/LicenseAllocationHeader.jsx

```diff
@@ -24,26 +17,18 @@ const LicenseAllocationHeader = () => {
         {' of '}
         {subscription.licenses?.total} licenses allocated
       </p>
-      <div className="my-3 row">
-        <div className="col-12 col-md-5 mb-3 mb-md-0">
+      <Row className="my-3">
+        <Col className="col-8">
```

Comment:
Does the search bar need to take up 8 columns when the user input is only really needs to be the length of a typical email address (I feel this will make the search bar take up more visual real estate than it needs)? Might recommend keeping it to 6 columns max for larger screens. 

Also, how does this look at smaller screen sizes since this will try to keep search bar and download button on the same row? I'm pretty sure we'd get a search bar that is not wide enough and a button than has some strange text wrapping behavior.

Previously, the search bar took up the full container width (12 columns) with the button left aligned below the search bar for small screens with the search bar collapsed to 5 columns for larger screens with the button right aligned in its column. I think we'll still want to keep that same sort of responsive behavior here.